lib/cocoapods/sandbox/path_list.rb

```diff
@@ -57,6 +57,11 @@ def read_file_system
         root_length = root.cleanpath.to_s.length + File::SEPARATOR.length
         Find.find(root.to_s) do |f|
           directory = File.directory?(f)
+          path = Pathname.new(f)
+          # File.symlink? does not detect folder symlinks
+          if File.symlink?(f) || (path.realpath != path)
```

Comment:
Would like a confirmation to make sure this logic is correct

Reply:
My guess is that only the latter part of the conditional is needed

Reply:
I think you're right, I had the left hand side for files but the latter should catch both files and directories

Reply:
I would also like to verify that this is needed at all? Since it looks like `find` may very well be traversing symlinks

Reply:
The attached project (on this PR) doesn't appear to be traversing symlinks